.github/workflows/ci.yml

```diff
@@ -480,20 +507,38 @@ jobs:
         with:
           cache-directories: ${{ env.CARGO_TARGET_DIR }}
 
-      - name: Rust Info
-        uses: ./.github/rust-info
-
       - name: Test Examples
         env:
           # Fix linking of `linkme`: https://github.com/dtolnay/linkme/issues/49
           RUSTFLAGS:                     -Clink-arg=-z -Clink-arg=nostart-stop-gc
-        run: |
+        uses: docker://paritytech/ci-unified:bullseye-1.75.0
```

Comment:
How to use here ${{ env.Image }} any hints?

Reply:
Can you elaborate on what you want to know?

Reply:
I would like to avoid duplicating image name and using the definition from the environment as `env.IMAGE`, but I cannot use the environment variable in this scope. Additionally, the workaround with `needs.set-image.outputs.IMAGE` does not work. Maybe there is another option which I do not know

Reply:
@alvicsam Do you know?